03/app.js

```diff
@@ -1,3 +1,14 @@
 const x = 10;
 let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let randomNumber = -1;
+
+while(randomNumber !== x) {
+    randomNumber = Math.round(Math.random() * 20);
+    iteration = 1;
+    iteration += iteration;
```

Comment:
Tutaj nie można wstawić ` iteration = 1` bo zawsze pokazuje, że wylosowaliśmy za 2 razem.
Wystarczy dodawać 1 zamiast to co jest w `iteration`, bo początkowo to 0, które nic nie zmiania ;)
Więc robimy: `iteration += 1;`

Reply:
Zadanie poprawione.